packages/babel-transforms/package.json

```diff
@@ -26,7 +26,7 @@
     "@babel/plugin-proposal-class-properties": "^7.1.0",
     "@babel/plugin-proposal-decorators": "^7.1.2",
     "ember-cli-babel-plugin-helpers": "^1.0.0",
-    "ember-cli-version-checker": "^2.1.0"
+    "ember-cli-version-checker": "^2.1.0 || ^3.0.0"
```

Comment:
Hmm, this is somewhat surprising. The main breaking change is dropping Node 4 which I think _we_ have already done so I think we can just use3 now...

Reply:
This is the result of me setting the version strategy of `peerDependencies` and `dependencies` to `"widen"`

https://github.com/ember-decorators/ember-decorators/blob/e9ed7cb174d8036678f08e19a8646dddf06f2624/renovate.json#L15-L21

I'll change this so that dependencies usually just replace the version (i.e., `^2.1.0` -> `^3`)

Reply:
Sounds good. I'm curious if there is a way to instruct renovate via a comment (inline or not) to replace instead of widen in the actual PRs?

Reply:
you mean like a comment in `package.json`?

Reply:
I think he meant like a way to tell the bot to change whatever it did in PR comments. Would be nice, but it looks like Renovate updates PRs when you update the config so no need I guess?

Reply:
Ya, I mean like if I commented inline on the PR with something akin to:

```
@renovate replace
```

Definitely not a big deal, just trying to get the hang of renovate...